source/guide_privatebin.rst

```diff
@@ -97,27 +99,27 @@ PrivateBin provides a .htaccess file, which blocks some known robots and link-sc
 
 Moving files outside of DocumentRoot
 ------------------------------------
-It is recommended_ to move the configuration, data files, templates and PHP libraries outside of your document root.
-To do that, create a folder  ``privatebin-data`` in ``/var/www/virtual/isabell/`` and move the folders to the new location (remember to replace ``isabell`` with your own username!).
+It is recommended_ to move the configuration, data files, templates and PHP libraries outside of your document root. This is useful to secure your installation.
+To do that, create a folder  ``privatebin-data`` in ``/home/isabell/`` and move the folders to the new location (remember to replace ``isabell`` with your own username!).
 If not already there, go to the ``html`` directory before running ``mv``.
 
 .. code-block:: console
 
  [isabell@stardust ~]$ cd ~/html
- [isabell@stardust html]$ mkdir /var/www/virtual/isabell/privatebin-data
```

Comment:
is there a reason for moving this directory?

Reply:
`/var/www/virtual/$USER/<folder>` is the DocumentRoot, i.e. the place where directories are located that are directly made accessible via the apache webserver (e.g. `html` or any other folder matching a subdomain, e.g. `privatebin.example.com`).

PrivateBin suggests to locate the data in a different directory than the public content, for access security reasons.
Uberspace manual also mentions  [1]:

> If one of the DocumentRoots gets compromised (e.g. because of a CVE), all other files within all other DocumentRoots can be compromised as well.

In all other guides, data is usually separated from DocumentRoot and located under `/home/$USER`. Thus, I wanted to follow this practice.

[1] https://manual.uberspace.de/web-documentroot/

Reply:
> In all other guides, data is usually separated from DocumentRoot and located under /home/$USER. Thus, I wanted to follow this practice.

Good idea, but shouldn't it be something like `~/privatebin-data` then?

Reply:
the choice of words in the manual is not very precise, I opened an Issue there to improve it. Actually, it is enough to store the data above the document roots in /var/www/username/ because only domain named folders and the html folders are interpreted as document roots. This is the way in all other guides. If there is a compromised PHP app, this will have full access to your uberspace anyway, that is why we propose to use a dedicated uberspace account for each app

Reply:
No, Apache can still access those directories, even without PHP. Granted, it would need something like an ill-configured `.htaccess`, but the possibility is there – which is why Nextcloud keeps it's data directory in $HOME.